native/go/Makefile

```diff
@@ -19,8 +19,10 @@ include ../../mk/spksrc.native-cc.mk
 
 myComp:
 	@$(MSG) "Building Go"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && ./make.bash
+	cd $(WORK_DIR)/$(PKG_NAME)/src && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash
 	@$(MSG) "Building Go for arm"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
 	@$(MSG) "Building Go for 386"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
+	@$(MSG) "Building Go for amd64"
```

Comment:
Go ahead and change. The README says we support x64, but that's not the same as actively preventing x86 from being used.
Iirc, Syncthing was the only package where it mattered which hostarch you were using, and you've largely removed the differences in this PR.
So if we can remove another difference by being explicit in what we're building, why not.

...assuming it works properly on both sides, but that should be easy to verify.